.gitignore

```diff
@@ -1,2 +1,4 @@
 /composer.lock
 /vendor
+/bin
+.idea
```

Comment:
I think that IDE-related instructions should be defined in a [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) as they are highly dependent on the users.